### DISCUSSION TITLE
NSI for etymology also?

### DISCUSSION THREAD

**Discussion by ivanbranco** [osmlab/name-suggestion-index#7493] [Ideas]
Hi, I saw that someone [already proposed](https://github.com/osmlab/name-suggestion-index/issues/6744) an entry with a name:etymology:wikidata in it, but the tag wasn't taken into account apparently. Why tho? I was thinking about [Zabka](https://nsi.guide/index.html?t=brands&k=shop&v=convenience&tt=abka#zabka-95f41b) (polish for "little frog") that could have name:etymology:wikidata=Q3116510 and [Biedronka](https://nsi.guide/index.html?t=brands&k=shop&v=supermarket&tt=biedr#biedronka-9e6a6c) (polish for "ladybug") that could have name:etymology:wikidata=Q25327. 

What do you think about this idea?

**Comment by 1ec5** [comment]
To the extent that an individual store location should have a `name` based on `brand`, the store location is directly named after the brand, not “frog” or “ladybug”. Have you considered adding a [named after (P138)](https://www.wikidata.org/wiki/Property:P138) statement to these brands’ Wikidata items? That would save a lot of work in terms of mappers having to add and maintain the etymology of the brand on each individual store location.

**Comment by LaoshuBaby** [comment]
For me I don't think this is a good idea because etymology isn't such important to a brand, and can't act as a identifier, it's a explanation to the `name`. We can't be a universal template.

What's more, some times we won't leave a default name, or use a `preserveTags:["^name.*"]` to avoid generate same name for every individual shop. In this situation mapper need to add many etymology manually for each one. (And you even can't say the etymology is for which part of the shop name, the brand part or branch part.)
